src/fairseq2/models/llama/tokenizer.py

```diff
@@ -191,7 +191,7 @@ def vocab_info(self) -> VocabularyInfo:
 
 @dataclass
 class LLaMATokenizerConfig:
-    impl: Literal["sp", "tiktoken", "hg"] = "sp"
+    impl: Literal["sp", "tiktoken", "hg"] = "hg"
```

Comment:
Just catching this up. Do you mind keeping this as `sp` so that we don't inadvertently break some dependent libraries? I think as long as we set "hg" as a "model_config" in the base LLaMA-3 asset card, this is not really needed anyways.